docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Adevinta
+
+version: "2.4"
```

Comment:
Remove obsolete `version` - https://github.com/compose-spec/compose-spec/blob/main/spec.md#version-and-name-top-level-elements

Reply:
I think it is obsolete if the build detects that can execute `docker compose`, but is mandatory if it has to execute `docker-compose`

Reply:
Ok then, not a big deal.

Reply:
Removed. I tested it with docker-compose v1.29.2, which is the version packaged by Fedora Linux 40, and it works just fine.